test/verify/check-shell-host-switching

```diff
@@ -183,6 +183,14 @@ class TestHostSwitching(testlib.MachineCase, HostSwitcherHelpers):
         b.wait_not_present("#nav-system.interact")
         b.set_layout("desktop")
 
+        # this looks different (focused) when the mouse is inside (default 0,0 possition), so move it away
+        # chromium is stubborn here, we *have* to move the real mouse, not just do this with synthetic MouseEvents
+        b.bidi("input.performActions", context=b.driver.context, actions=[{
```

Comment:
Only affects chrome, then we could gate it? But lets get CI green again

Reply:
I could make it conditional on `.browser == chromium`, but we don't run pixel tests on Firefox anyway, so it's no practical difference.